test/tap/NOISE_TESTING.md

```
@@ -0,0 +1,106 @@
+
+Initial noise scripts are provided in `test/tap/noise/`:
+
+1.  **`noise_stats_poller.py`**: Periodically queries `stats_mysql_query_digest` and `stats_mysql_connection_pool`.
+2.  **`noise_admin_pinger.sh`**: Executes `SELECT 1` against the Admin interface.
+3.  **`noise_pgsql_poller.sh`**: Generates simple PostgreSQL traffic using `psql`.
+
+## Built-in Noise Routines (Internal)
+
+Internal routines run as threads within the TAP test process. This is **highly recommended for debugging with GDB**, as stopping the test process will also pause the noise.
+
+#### Standard Internal Noise Functions:
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)